pkg/madmin/remote-target-commands.go

```diff
@@ -235,8 +237,43 @@ func (adm *AdminClient) SetRemoteTarget(ctx context.Context, bucket string, targ
 	return arn, nil
 }
 
+// TargetUpdateType -  type of update on the remote target
+type TargetUpdateType int
+
+const (
+	CredentialsUpdateType TargetUpdateType = 1 + iota
+	SyncUpdateType
+	ProxyUpdateType
+	BandwidthLimitUpdateType
+	HealthCheckDurationUpdateType
+	PathUpdateType
+)
+
+func GetTargetUpdateOps(u *url.URL) []TargetUpdateType {
```

Comment:
```suggestion
func GetTargetUpdateOps(u *url.URL) (ops []TargetUpdateType) {
```

Please add a comment and unit test @poornas